src/components/withWindowDimensions.js

```diff
@@ -46,11 +49,14 @@ export default function (WrappedComponent) {
             const isMediumScreenWidth = initialDimensions.width > variables.mobileResponsiveWidthBreakpoint
               && initialDimensions.width <= variables.tabletResponsiveWidthBreakpoint;
 
+            const isLandscape = (isMediumScreenWidth || isSmallScreenWidth) && initialDimensions.width >= initialDimensions.height;
```

Comment:
I get that checking if the width is greater than the height could tell us if we are in landscape. But why is the screen width relevant?

Reply:
Wanted to keep the flag enabled only for Tablets and Mobile by default. Will remove.

Reply:
Got it. If we want to do that does it make sense let's check the size of the screen wherever we are using this? 
Not sure I would expect an `isLandscape` prop to return `false` if we are past a certain width. We either have something oriented with its length shorter than width or we don't.

Reply:
Noted. I've removed the screen width check. I guess `<Modal type={props.isSmallScreenWidth ? CONST.MODAL.MODAL_TYPE.BOTTOM_DOCKED : CONST.MODAL.MODAL_TYPE.POPOVER}` should cover for the screen width check that I need.